### PR TITLE
Fix image picker

### PR DESCRIPTION
**Pull request by noirbizarre**
This pull-request fix mutualize image upload handling, fix image picker and ModelList.

### CODE REVIEW

File: udata/core/storages/api.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+def parse_uploaded_image(field):
+    '''Parse an uploaded image and save into a db.ImageField()'''
+    args = image_parser.parse_args()
+
+    image = args['file']
+    bbox = args.get('bbox', None)
+    if bbox:
+        bbox = [int(float(c)) for c in bbox.split(',')]
```
Comment (davidbgk) [inline udata/core/storages/api.py]: `int` is not enough?

File: udata/forms/fields.py [hunk 2]
```diff
@@ -370,15 +370,11 @@ def _value(self):
     def process_formdata(self, valuelist):
         if not valuelist:
             return []
-        if len(valuelist) > 1:
-            oids = [clean_oid(id, self.model) for id in valuelist]
-        elif isinstance(valuelist[0], basestring):
-            oids = [
-                clean_oid(id, self.model)
-                for id in valuelist[0].split(',') if id]
+        if len(valuelist) == 1 and isinstance(valuelist[0], basestring):
```
Comment (davidbgk) [inline udata/forms/fields.py]: `if valuelist` looks to be enough or I miss a case?

**Comment by davidbgk** [comment]
r+

File: udata/core/storages/api.py (hunk 1 above)
Comment (noirbizarre) [inline udata/core/storages/api.py]: float to cast from string to float, int to round to the pixel.

**Pull request merged**